lib/model/query/forms.js

```diff
@@ -393,14 +397,33 @@ ${extend|| sql`
   left outer join actors on audits."actorId"=actors.id
   left outer join (select id, "contentType" as "excelContentType" from blobs) as xls
     on form_defs."xlsBlobId"=xls.id`}
+${(actorId == null) ? sql`` : sql`
+inner join
+  (select id, max(assignment."showDraft") as "showDraft" from projects
```

Comment:
I think you could also make `showDraft` a boolean, then use `bool_or(assignment."showDraft")` instead of `max()`.